packages/preview/src/browser/preview-handler.ts

```diff
@@ -31,15 +40,67 @@ export namespace RenderContentParams {
     }
 }
 
+/**
+ * A PreviewHandler manages the communication between text editor and preview widget including whether a preview shall be shown at all.
+ * 
+ * See {@link MarkdownPreviewHandler} for an example implementation.
+ */
 export interface PreviewHandler {
+    /**
+     * One or more classes which shall be applied to the preview widget icon.
+     */
     readonly iconClass?: string;
+    /**
+     * Indicates whether and with which priority (larger is better) this preview handler is responsible for the given resource identified by the given URI.
+     * 
+     * @param uri the URI identifying a resource.
+     * 
+     * @returns A number larger than 0 if the handler is applicable, 0 or a negative number otherwise.
+     */
     canHandle(uri: URI): number;
+    /**
+     * Render the preview content by returning appropriate HTML.
+     * 
+     * @param params Information for the handler to render its content. Contains at least the textual content and the document URI.
```

Comment:
I believe we can remove "contains at least the textual content and the document URI".
This is clear based on the typings of `RenderContentParams` and is subject to change.